data-model/SDP-sample-records/work_sample1.json

```diff
@@ -0,0 +1,34 @@
+{
+  "id": "https://sinaimanuscripts.library.ucla.edu/work/10",
+  "label": "Hexaemeron (Basil of Caesarea)",
+  "entity_type": "work",
+  "uniform_name": "Basil, Saint, Bishop of Caesarea, approximately 329-379. Hexaemeron",
+  "description": "A commentary on the six days of Creation. The first of its kind, this style of commentary, called Hexaemeron, flourished in late antiquity, and later commentaries drew on Basil's as a model.",
+  "authority_files": [{
+      "id": "https://haf.vhmml.org/work/320756617003",
+      "uniform_title": "Basil, Saint, Bishop of Caesarea, approximately 329-379. Hexaemeron"
+    },
+    {
+      "id": "https://viaf.org/viaf/181934772",
+      "uniform_title": "Basil, Saint, Bishop of Caesarea, approximately 329-379. | Hexaemeron"
+    }
+  ],
```

Comment:
How do we add authority info that doesn’t have associated URIs? CPG, BHO, etc. numbers (though CPG does technically have the OA Clavis Clavium, which I think has a unique URL for each entity, though it may not be stable). Do we establish a controlled abbreviation schema? Do we have a separate ‘reference numbers’ field? I think we’d want to consider these authority files still?